Makefile

```diff
@@ -70,6 +70,7 @@ post-install:
 	@cp ./native_modules/ImagePickerModule.java node_modules/react-native-image-picker/android/src/main/java/com/imagepicker
 	@# Need to copy custom RNDocumentPicker.m that implements direct access to the document picker in iOS
 	@cp ./native_modules/RNDocumentPicker.m node_modules/react-native-document-picker/ios/RNDocumentPicker/RNDocumentPicker.m
+	@cp ./native_modules/react-native-youtube/* node_modules/react-native-youtube
```

Comment:
I'm doing this just to copy over the modified files to add the functionality for iOS instead of forking the whole project, let me know if any of you prefer to fork the project instead

Reply:
Are these changes we could contribute back to them? It would be nice to not have to modify these in the Makefile

Reply:
Yea we can but I do not want to wait until they merge it If they want to take it

Reply:
That's fine, but that would require us to fork it anyway 😉. 0/5 on if we actually use the fork or not though

Reply:
PR for this https://github.com/inProgress-team/react-native-youtube/pull/294

I'm going to update this PR to use the fork for now